src/components/calcite-tile-select/calcite-tile-select.tsx

```diff
@@ -67,6 +67,27 @@ export class CalciteTileSelect {
 
   private input: HTMLCalciteCheckboxElement | HTMLCalciteRadioButtonElement;
 
+  //--------------------------------------------------------------------------
+  //
+  //  Private Methods
+  //
+  //--------------------------------------------------------------------------
+
+  private checkFirstTileSelect(): void {
+    const tileSelects = document.querySelectorAll(`calcite-tile-select[name=${this.name}]`);
+    let firstCheckedTileSelect: HTMLCalciteTileSelectElement;
+    if (tileSelects && tileSelects.length > 0) {
+      tileSelects.forEach((tileSelect: HTMLCalciteTileSelectElement) => {
+        if (firstCheckedTileSelect) {
+          tileSelect.checked = false;
+        } else if (tileSelect.checked) {
+          firstCheckedTileSelect = tileSelect;
+        }
+        return tileSelect;
+      });
+    }
+  }
```

Comment:
This functionality is identical to how `radio-button` does this same thing for other radios of the same name.  For some reason (not sure what changed) but `radio-button`'s built-in check first behavior stopped working for `tile-select` groups.  It seems to be related to some kind of race condition because `querySelectorAll` returns an empty NodeList for `radio-button` elements.  Must have something to do with the mounting order of child components or something. 🤷